src/puppy/common.nim

```diff
@@ -1,35 +1,30 @@
 import urlly, strutils
-
```

Comment:
Could these blank lines / formatting be put back? It's a good idea to not change more than is necessary in a file.